ansible_collections/arista/avd/roles/eos_designs/docs/input-variables.md

```diff
@@ -179,12 +179,14 @@ A stale entry is an entry that was not accessed during the run.
 
     To reuse a freed ID, first remove the old device and run AVD. Then add the new device and rerun AVD.
 
-The pool manager stores data in a YAML file per fabric. The default path is `<root_dir>/intended/data/<fabric_name>-ids.yml`
-
 !!! tip
     It is possible to override the automatic assignments by editing the data files manually.
     Just make sure to have a backup or use source control like Git and rerun AVD after changing the file.
 
+--8<--
+ansible_collections/arista/avd/roles/eos_designs/docs/tables/fabric-numbering.md
```

Comment:
This is already in line 153, but I can see the new path style was missed on that one.

Reply:
Right, I missed it... I moved to the top section as well, and I think that makes the most sense.
I'm good either way, though, so happy to move it back!